CMakeLists.txt

```diff
@@ -966,7 +966,7 @@ add_library_variant(
 add_library_variant(
     armv6m
     SUFFIX soft_nofp
-    COMPILE_FLAGS "-mfloat-abi=soft -march=armv6m"
+    COMPILE_FLAGS "-mfloat-abi=soft -march=armv6-m"
```

Comment:
I'd prefer these changes weren't made. `armv6m` works with both `-march` and `--target` whereas `armv6-m` only works with the former.

Reply:
Ok. Reverting. The motivation was to be in line with GCC documentation.